benches/iai-callgrind/benchmarks.cfg

```diff
@@ -0,0 +1,4 @@
+json_like_bench_iai_callgrind::batched_body::benchmark_batched_body
+data_loader_bench_iai_callgrind::data_loader::benchmark_data_loader
+impl_path_string_for_evaluation_context_iai_callgrind::bench::bench_main
+request_template_bench_iai_callgrind::bench_to_request::benchmark_to_request
```

Comment:
We should not have to update this file everytime we add a benchmark. We are anyways updating `cargo.toml`. This should not be used or autogenerated somehow.